misc/python/materialize/mzcompose/services.py

```diff
@@ -170,10 +170,16 @@ def __init__(
     ) -> None:
         environment = [
             "CLUSTERD_LOG_FILTER",
-            "MZ_SOFT_ASSERTIONS=1",
```

Comment:
IIRC the environment variables are defined in that order, meaning that you can just keep 
```bash
MZ_SOFT_ASSERTIONS=1
```
here and override with
```
MZ_SOFT_ASSERTIONS=0
```
in `environment_extra`. For example, see how the `replica-isolation` test overrides the `CLUSTERD_LOG_FILTER`.

Reply:
Sounds great, thanks!